--- origsrc/ltris-1.0.12/src/Makefile.am	2005-10-05 08:32:52.000000000 -0500
+++ src/ltris-1.0.12/src/Makefile.am	2008-04-15 08:50:10.453125000 -0500
@@ -4,7 +4,7 @@
 ltris_SOURCES = sdl.c menu.c item.c manager.c value.c main.c audio.c event.c \
 list.c tools.c config.c chart.c tetris.c bowl.c shrapnells.c cpu.c hint.c \
 parser.c
-ltris_LDADD   = @mixer_flag@
+ltris_LDADD   = @mixer_flag@ $(LTLIBINTL)
 
 EXTRA_DIST = sdl.c menu.c item.c manager.c value.c main.c audio.c event.c \
 list.c tools.c config.c chart.c tetris.c bowl.c shrapnells.c cpu.c \
@@ -17,10 +17,10 @@
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)
-	$(INSTALL_DATA) figures $(DESTDIR)$(inst_dir)/figures
+	$(INSTALL_DATA) $(srcdir)/figures $(DESTDIR)$(inst_dir)/figures
 	$(mkinstalldirs) $(DESTDIR)$(hi_dir)
 	if ! test -f $(DESTDIR)$(hi_dir)/ltris.hscr; then \
-	  $(INSTALL_DATA) --mode=666 empty.hscr $(DESTDIR)$(hi_dir)/ltris.hscr; \
+	  $(INSTALL_DATA) --mode=666 $(srcdir)/empty.hscr $(DESTDIR)$(hi_dir)/ltris.hscr; \
 	fi;
 
 uninstall-local:
--- origsrc/ltris-1.0.12/src/gfx/Makefile.am	2002-01-09 14:35:12.000000000 -0600
+++ src/ltris-1.0.12/src/gfx/Makefile.am	2008-04-15 08:46:52.406250000 -0500
@@ -4,6 +4,6 @@
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/gfx
-	@for file in *.bmp; do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/gfx/$$file;\
+	@for file in $(EXTRA_DIST); do\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/gfx/$$file;\
 	done
--- origsrc/ltris-1.0.12/src/sounds/Makefile.am	2002-01-07 14:30:24.000000000 -0600
+++ src/ltris-1.0.12/src/sounds/Makefile.am	2008-04-15 08:46:52.406250000 -0500
@@ -3,6 +3,6 @@
 
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(inst_dir)/sounds
-	@for file in *.wav; do\
-	  $(INSTALL_DATA) $$file $(DESTDIR)$(inst_dir)/sounds/$$file;\
+	@for file in $(EXTRA_DIST); do\
+	  $(INSTALL_DATA) $(srcdir)/$$file $(DESTDIR)$(inst_dir)/sounds/$$file;\
 	done
